cura/CuraApplication.py

```diff
@@ -348,57 +351,19 @@ def __init__(self, **kwargs):
 
         preferences.setDefault("local_file/last_used_type", "text/x-gcode")
 
-        preferences.setDefault("general/visible_settings", """
```

Comment:
Huge +1 for removing these lines :P

Reply:
Huge +1 x2

Reply:
Oh, it's my PR... Somehow I thought this is Ian's....

Reply:
I'll take it.